packages/create-fuels/src/cli.ts

```diff
@@ -13,13 +14,49 @@ const log = (...data: unknown[]) => {
   process.stdout.write(`${data.join(' ')}\n`);
 };
 
+type ProgramsToInclude = {
+  contract: boolean;
+  predicate: boolean;
+  script: boolean;
+};
+
+const processWorkspaceToml = (fileContents: string, programsToInclude: ProgramsToInclude) => {
+  const parsed = toml.parse(fileContents) as {
+    workspace: {
+      members: string[];
+    };
+  };
+
+  parsed.workspace.members = parsed.workspace.members.filter((member) => {
+    if (member === 'predicate' && !programsToInclude.predicate) {
+      return false;
+    }
+    if (member === 'contract' && !programsToInclude.contract) {
+      return false;
+    }
+    if (member === 'script' && !programsToInclude.script) {
+      return false;
+    }
+    return true;
```

Comment:
```suggestion
    switch (member) {
      case 'predicate':
        return programsToInclude.predicate;
      case 'contract':
        return programsToInclude.contract;
      case 'script':
        return programsToInclude.script;
      default:
        return true;
    }
```

Reply:
If line 26 changes from this:

```ts
members: string[];
```

To this:

```ts
members: ('predicate' | 'contract' | 'script')[];
```

Could we get away with just this?

```ts
parsed.workspace.members = parsed.workspace.members.filter((m) => programsToInclude[m]);
```

Reply:
Yeah, I was thinking about that too but didn't suggest it just because if any of the property names of `ProgramsToInclude` changes or if the members in `Forc.toml` change the code will break. I'm fine with that suggestion as well, though, as I feel this concern of mine isn't that big of a deal.

Reply:
I'm not sure if this could make it more resilient.

```ts
parsed.workspace.members = parsed.workspace.members.filter((m) => programsToInclude[m] ?? true);
```

Reply:
On second thought, we can safely go with
```ts
parsed.workspace.members = parsed.workspace.members.filter((m) => programsToInclude[m]);
```
if the tests cover it properly.  They should be covering it in one way or another anyways. I added a comment related to these tests [here](https://github.com/FuelLabs/fuels-ts/pull/1691/files#r1478031625).

Reply:
Code-wise, the first version looks just fine; it is a trivial change that may not even deserve this much attention.